src/sentry/release_health/metrics.py

```diff
@@ -122,7 +123,83 @@ def _model_environment_ids_to_environment_names(
 
 
 class MetricsReleaseHealthBackend(ReleaseHealthBackend):
-    """Gets release health results from the metrics dataset"""
+    """
+    Implementation of the ReleaseHealthBackend using the MetricsLayer API
+    """
+
+    @staticmethod
+    def _get_org_id(project_ids: Sequence[int]) -> int:
+        return MetricsReleaseHealthBackend._get_projects_and_org_id(project_ids)[1]
```

Comment:
do we want to check IndexError, or it's always have length == 2?

Reply:
`_get_projects_and_org_id()` returns `Tuple[Sequence[Project], int]`
so `_get_projects_and_org_id(projects)[1]` returns the second element of the tuple, i.e. the `organisation_id`
`_get_projects_and_org_id(projects)[0]` returns the `Sequence[Project]`

The reason we have `get_projects_and_org_id` as one function is that obtaining both the Sequence[Project] and the org_id from Sequence[proj_id] requires getting the Project models from the cache and since we often need both at the same time we don't want to do two lookups. 

... since the logic is a bit obscure and error prone I did the small utility functions above that take care of this "nastiness" `_get_org_id` and `_get_projects` which extract the two elements from the tuple and don't bother the user with the details.

Reply:
good point, resolved.